Makefile

```diff
@@ -1,4 +1,6 @@
-include config.mk
+# The "-" in front of the import is to ignore errors if the file doesn't exist.
+# It lets us run `make` without having to create `config.mk` first:
+-include config.mk
```

Comment:
The configure.mk file is used for 2 things one for defining installation paths, and for defining the xapi version. This is most relevant when building the rpm package, so I don't think it's critical to make it error when it's not present for developers.

Any other ideas? @edwintorok 
Otherwise I'm fine with it